Mobile/lib/pages/login.dart

```diff
@@ -24,69 +26,86 @@ class _LoginState extends State<Login> {
     _appState = Provider.of<RootAppState>(context);
 
     return Scaffold(
-      appBar: AppBar(
-        title: Text('Login'),
-        backgroundColor: _theme.colorScheme.primaryContainer,
-      ),
       backgroundColor: _theme.colorScheme.primaryContainer,
-      body: Padding(
-        padding: const EdgeInsets.all(20),
-        child: Form(
-          key: _formKey,
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              TextFormField(
-                decoration: InputDecoration(labelText: 'username / email'),
-                validator: (value) {
-                  if (value == null || value.isEmpty) {
-                    return 'Username is required and cannot be empty';
-                  }
-                  return null;
-                },
-                onSaved: (value) {
-                  _username = value!;
-                },
-              ),
-              SizedBox(height: 10),
-              TextFormField(
-                decoration: InputDecoration(labelText: 'password'),
-                obscureText: true,
-                validator: (value) {
-                  if (value == null || value.isEmpty) {
-                    return 'Password is required and cannot be empty';
-                  }
-                  return null;
-                },
-                onSaved: (value) {
-                  _password = value!;
-                },
-              ),
-              SizedBox(height: 15),
-              ElevatedButton(
-                onPressed: tryLogin,
-                child: Text('Login'),
-              ),
-              Container(
-                height: 2,
-                width: 150,
-                color: Colors.grey,
-                margin: EdgeInsets.symmetric(vertical: 8),
-              ),
-              ElevatedButton(
-                onPressed: () {
-                  Navigator.push(
-                    context,
-                    MaterialPageRoute(
-                      builder: (context) => Register(),
+      body: Stack(
+        children: [
+          MeshGradientBackground(),
+          Padding(
+            padding: const EdgeInsets.all(20),
+            child: Column(
+              children: [
+                // Logo
+                Image.file(
+                  File('assets\\Logo.png'),
+                  height: 150,
+                  fit: BoxFit.cover,
+                ),

```

Comment:
Image.file doesn't seem to work correctly on an android phone.
I suggest using Image.asset instead as to my testing it works better.
I would also suggest using `/` instead of `\\` as double backslash is only on windows, most other systems uses a single forward slash.
```suggestion
                Image.asset(
                  'assets/Logo.png',
                  height: 150,
                  fit: BoxFit.cover,
                ),

```